CHANGES.rst

```diff
@@ -1,5 +1,7 @@
 Upcoming version
 ================
+Main items of this new release:
+- Added support for numpy 2.0; limited the allowed numpy version to <3.
```

Comment:
Should we mention the upper limit of numpy?

Reply:
Maybe not, I removed the remarks.